### ISSUE TITLE
Installing of Pil fails

### ISSUE THREAD

**Issue by pardo-bsso** [OCA/maintainer-quality-tools#433]
Hello all and thanks for this tool.

We are integrating Travis in one of our modules and the travis_install_nightly fails when tries to install Pil.
Did anybody else have this problem? This is an extract of the build log:

    Installing Odoo https://github.com/odoo/odoo/archive/9.0.tar.gz
    
    2017-02-22 21:04:57 URL:https://codeload.github.com/odoo/odoo/tar.gz/9.0 [123010605] -> "odoo.tar.gz" [1]
    
    Collecting pip from https://pypi.python.org/packages/b6/ac/7015eb97dc749283ffdec1c3a88ddb8ae03b8fad0f0e611408f196358da3/pip-9.0.1-py2.py3-none-any.whl#md5=297dbd16ef53bcef0447d245815f5144
    
      Downloading pip-9.0.1-py2.py3-none-any.whl (1.3MB)
    
        100% |################################| 1.3MB 434kB/s
    
    Installing collected packages: pip
    
      Found existing installation: pip 6.0.7
    
        Uninstalling pip-6.0.7:
    
          Successfully uninstalled pip-6.0.7
    
    Successfully installed pip-9.0.1
    
    /home/travis/virtualenv/python2.7_with_system_site_packages/local/lib/python2.7/site-packages/pip/_vendor/requests/packages/urllib3/util/ssl_.py:318: SNIMissingWarning: An HTTPS request has been made, but the SNI (Subject Name Indication) extension to TLS is not available on this platform. This may cause the server to present an incorrect TLS certificate, which can cause validation failures. You can upgrade to a newer version of Python to solve this. For more information, see https://urllib3.readthedocs.io/en/latest/security.html#snimissingwarning.
    
      SNIMissingWarning
    
    /home/travis/virtualenv/python2.7_with_system_site_packages/local/lib/python2.7/site-packages/pip/_vendor/requests/packages/urllib3/util/ssl_.py:122: InsecurePlatformWarning: A true SSLContext object is not available. This prevents urllib3 from configuring SSL appropriately and may cause certain SSL connections to fail. You can upgrade to a newer version of Python to solve this. For more information, see https://urllib3.readthedocs.io/en/latest/security.html#insecureplatformwarning.
    
      InsecurePlatformWarning
    
    Will not install to the user site because it will lack sys.path precedence to Pillow in /home/travis/virtualenv/python2.7_with_system_site_packages/lib/python2.7/site-packages
    
    The command "travis_install_nightly" failed and exited with 1 during .
    
    Your build has been stopped.

**Comment by pardo-bsso** [comment]
Leaving pillow without a version in requirements.txt worked to overcome this.
Is there any specific reason to use that version?

**Comment by moylop260** [comment]
You need follow the manual to install dependencies:
 - https://www.odoo.com/documentation/9.0/setup/install.html

**Comment by pardo-bsso** [comment]
Thanks for your time, perhaps I explained myself bad.

I followed the instructions detailed here: https://github.com/OCA/maintainer-quality-tools/blob/master/README.md

In order to have Travis build and test our module against odoo 9 I had to change this file: https://github.com/OCA/maintainer-quality-tools/blob/master/travis/requirements.txt , reverting your last commit.

Anyways, I managed to make it work. Thanks!
